js/src/carousel.js

```diff
@@ -230,23 +261,74 @@ class Carousel {
       $(this._element)
         .on(Event.MOUSEENTER, (event) => this.pause(event))
         .on(Event.MOUSELEAVE, (event) => this.cycle(event))
-      if ('ontouchstart' in document.documentElement) {
+    }
+
+    this._addTouchEventListeners()
+  }
+
+  _addTouchEventListeners() {
```

Comment:
beyond adding the event listener, this code should also add `touch-action: pan-x` to the carousel when pointer events are used ... `preventDefault()` doesn't stop the forward/back browser navigation for pointer events, it's done declaratively with `touch-action` in the PE model. either doing it dynamically (adding a `style`? seems a bit dirty) or adding it to the definition of carousel CSS styles (though then it would "swallow" horizontal swipes in a PE browser even if the author DIDN'T want touch to be handled on a carousel)

Reply:
I added a class when we use PointerEvents but I don't understand why it doesn't work on IE 10/11, do you any ideas why @patrickhlauke ? 🤔

Reply:
i'm an idiot...and suggested the wrong `touch-action` value...i should go and have a word with the co-editor and chair of the PE spec, it seems...